curves/bn254/src/lib.rs

```diff
@@ -198,12 +199,23 @@ mod target_arch {
     }
 
     pub fn alt_bn128_multiplication(input: &[u8]) -> Result<Vec<u8>, AltBn128Error> {
-        if input.len() > ALT_BN128_MULTIPLICATION_INPUT_LEN {
+        alt_bn128_apply_multiplication(input, ALT_BN128_MULTIPLICATION_INPUT_LEN_FIX)
+    }
+
+    pub fn alt_bn128_multiplication_128(input: &[u8]) -> Result<Vec<u8>, AltBn128Error> {
+        alt_bn128_apply_multiplication(input, ALT_BN128_MULTIPLICATION_INPUT_LEN)
+    }
```

Comment:
```suggestion
        alt_bn128_apply_multiplication(input, ALT_BN128_MULTIPLICATION_INPUT_LEN)
    }

    pub fn alt_bn128_multiplication_128(input: &[u8]) -> Result<Vec<u8>, AltBn128Error> {
        alt_bn128_apply_multiplication(input, 128) // hard-code length; we will remove this function in the future
    }
```
Let's just hardcode the value 128 in here instead of a defining a new constant. We will be removing this function eventually.